ert/data/record/_transformation.py

```diff
@@ -20,6 +20,11 @@ def _prepare_location(base_path: Path, location: Path) -> None:
 
 
 class RecordTransformation(ABC):
+    """:class:`RecordTransformation` is an abstract class that handles
+    custom operations on Records or disk making use of :class:`RecordTransmitter`
+    API.
+    """
```

Comment:
Can either the `"custom operations"` or the corresponding "domain" of `RecordTransformation` be specified (e.g., it handles record serialization++ operations, with ++ being clarified)?

Reply:
Can rewrite it as follows: 
""":class:`RecordTransformation` is an abstract class that handles
    custom operations, ie. :class:`FileRecordTransformation`, :class:`TarRecordTransformation` and :class:`ExecutableRecordTransformation`; on Records or disk making use of :class:`RecordTransmitter` API.
    """